dkg/dkg.go

```diff
@@ -25,16 +24,32 @@ type Actor interface {
 	// setup has not been done.
 	GetPublicKey() (kyber.Point, error)
 
+	// Encrypt encrypts the given message into kyber points
+	// using the DKG public key
 	Encrypt(message []byte) (K, C kyber.Point, remainder []byte, err error)
+	// Decrypt decrypts a pair of kyber points into the original message
+	// using the DKG internal private key
 	Decrypt(K, C kyber.Point) ([]byte, error)
 
+	// Reshare recreates the DKG with an updated list of participants.
 	Reshare(co crypto.CollectiveAuthority, newThreshold int) error
 
+	// VerifiableEncrypt encrypts the given message into kyber points
+	// using the DKG public key and proof of work algorithm.
 	VerifiableEncrypt(message []byte, GBar kyber.Point) (ciphertext types.Ciphertext, remainder []byte, err error)
+
+	// VerifiableDecrypt decrypts a pair of kyber points into the original message
+	// using the DKG internal private key and a proof of work algorithm.
 	VerifiableDecrypt(ciphertexts []types.Ciphertext) ([][]byte, error)
```

Comment:
Proof-of-Work ? Are you sure about this one ? 😅 
My understanding is that we're just creating a zero-knowledge proof.

Reply:
reworded